lib/compat/wordpress-6.4/script-loader.php

```diff
@@ -107,3 +107,109 @@ static function ( $settings ) {
 		return $settings;
 	}
 );
+
```

Comment:
Should this be in the 6.5 folder though? This fix is not on 6.4 right?

Reply:
That's true. I mislead with being compatible with 6.4 versions.